extensions/porkbun/src/domain-pricing.tsx

```diff
@@ -97,10 +97,11 @@ export default function DomainPricing() {
       navigationTitle={navigationTitle}
       actions={
         <ActionPanel>
-          <Action icon={Icon.Redo} title="Reload Domain Pricing" onAction={callApi} />
+          {!isLoading && <Action icon={Icon.Redo} title="Reload Domain Pricing" onAction={callApi} />}
           <Action.OpenInBrowser icon={Icon.Globe} title="Go to API Reference" url={`${API_DOCS_URL}Domain%20Pricing`} />
         </ActionPanel>
       }
+      searchBarPlaceholder="Search domain pricing"
```

Comment:
To be more specific should we use this:
```suggestion
      searchBarPlaceholder="Search domain by name"
```